zhmcclient/_session.py

```diff
@@ -637,3 +635,20 @@ def delete_completed_job_status(self, job_uri):
           :exc:`~zhmcclient.ConnectionError`
         """
         self.delete(job_uri)
+
+    @_log_call
+    def get_notfication_topics(self):
```

Comment:
Typo: get_**notfication**_topics() should changed to get_**notification**_topics().